app/src/main/java/de/xikolo/App.kt

```diff
@@ -102,4 +117,29 @@ class App : Application() {
         CookieManager.getInstance().flush()
     }
 
+    private fun updateShortcuts() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            val shortcutManager = getSystemService<ShortcutManager>(ShortcutManager::class.java)
+            val recentCourses = RecentCoursesStorage().recentCourses
+            val intentList = mutableListOf<ShortcutInfo>()
+
+            if (recentCourses != null) {
+                for (course in recentCourses) {
```

Comment:
Isn't that call `null` safe already? (Not sure, pls find out.)

Reply:
Yes, it is